### ISSUE TITLE
Search help on focused word got broken

### ISSUE THREAD

**Issue by pgoral** [godotengine/godot#28672]
**Godot version:** 84401e8cdf4f41fc0e804bf5410622e75a08c0f5

**Issue description:**
Previously (on 3.1.1 stable release) when I did Shift+F1 I got search help with filled word that I had focus on. Now it doesn't work that way. Search field is empty.

I deleted AppData Godot folder, but it didn't help

**Comment by volzhs** [comment]
confirmed. it's really annoying regression.

**Comment by pgoral** [comment]
Also, when you run 84401e8 version, and then switch back to 3.1.1 it also stops working correctly. You have to delete Godot folder from AppData to restore that functionality on 3.1.1

**Comment by volzhs** [comment]
i don't know why godot's appdata is related but sounds strange to me. hm.

**Comment by pgoral** [comment]
I have just rechecked this. And yes, you have to delete godot folder

**Comment by YeldhamDev** [comment]
Does it work normally before my commit? Because analysing it here, it doesn't seem to checks for highlighted words:
https://github.com/godotengine/godot/blob/84401e8cdf4f41fc0e804bf5410622e75a08c0f5/editor/plugins/script_editor_plugin.cpp#L1010-L1013

**Comment by YeldhamDev** [comment]
Never mind, that was for the global shortcut, which seems to be overriding the script editor one, I don't know how my commit did that...

**Comment by YeldhamDev** [comment]
I reverted the commit on my end, cleared my "godot" directory in ".config", and the help still ignores highlighted text. I don't think this is related to my commit at all.

**Comment by YeldhamDev** [comment]
I think I found the problem! In the "editor_settings-3.tres" file generated in `master`, this piece of text is added:
```
[sub_resource type="InputEventKey" id=1]
shift = true
scancode = 16777244
unicode = 16777244
```

If you remove this and launch the 3.1.* version, the help dialog grabs the highlighted text as expected. However, I don't know what triggers to add that there.

**Comment by dankan1890** [comment]
Related #27151

**Comment by akien-mga** [comment]
That's a regression from f1e4fd3e39.

There are now two locations where this shortcut is defined to different values:
```
5854:   p->add_icon_shortcut(gui_base->get_icon("HelpSearch", "EditorIcons"), ED_SHORTCUT("editor/editor_help", TTR("Search"), KEY_MASK_SHIFT | KEY_F1), HELP_SEARCH);
6406:   ED_SHORTCUT("editor/editor_help", TTR("Search Help"), KEY_F4);
```

The shortcut should also be changed on line 6406, or f1e4fd3e39 should be reverted.

**Comment by akien-mga** [comment]
Nevermind my comment, that's already what @dankan1890 had found out and fixed in #28907.

**Comment by pgoral** [comment]
Why this has been closed? It still doen't work. The #28907 didn't resolved it.
